Heron/ImportVector.cs

```diff
@@ -53,9 +53,8 @@ protected override void SolveInstance(IGH_DataAccess DA)
             List<Curve> boundary = new List<Curve>();
             DA.GetDataList<Curve>("Boundary", boundary);
 
-            string shpFileLoc = "";
-            DA.GetData<string>("Vector Data Location", ref shpFileLoc);
-
+            string shpFilePath = "";
```

Comment:
it can be good to be explicit with things like these using the string alias (ie string.Empty) to make it more immediately clear that it is an empty string

Reply:
Agreed.  This way of declaring an empty string happens a lot in the code.  We will need to keep string.Empty in mind as we update the code in other places.